--- taglist.vim.orig    2015-06-23 13:09:39.000000000 +0100
+++ taglist.vim 2015-06-24 13:20:39.000000000 +0100
@@ -471,6 +471,9 @@
 " perl language
 let s:tlist_def_perl_settings = 'perl;c:constant;l:label;p:package;s:subroutine'
 
+" perl6 language
+let s:tlist_def_perl6_settings = 'perl6;c:constant;l:label;p:package;s:subroutine'
+
 " php language
 let s:tlist_def_php_settings =
             \ 'php;c:class;i:interface;d:constant;v:variable;f:function'

